compendium/DeclarativeServices/src/manager/states/CCActiveState.hpp

```diff
@@ -82,9 +83,15 @@ class CCActiveState final : public CCSatisfiedState
    * state represented by this object
    */
   ComponentState GetValue() const override { return ComponentState::ACTIVE; }
+  void WaitForTransitionTask() override {
+      latch.Wait(); 
+  }
+  void WaitForStateChangeCompletion() override { return; }
```

Comment:
I don't understand the purpose of `WaitForStateChangeCompletion()`, can you elaborate? What is this function's purpose?